tests/lifecycle/init_resp.json

```diff
@@ -25,6 +25,22 @@
             "inlayHintProvider": true,
             "referencesProvider": true,
             "renameProvider": true,
+            "semanticTokensProvider": {
+                "full": true,
+                "range": true,
+                "legend": {
+                    "tokenModifiers": [
+                        "definition"
+                    ],
+                    "tokenTypes": [
+                        "variable",
+                        "function",
+                        "string",
+                        "number",
+                        "regexp"
```

Comment:
It seems we better generate this JSON via `*semantic-token-types*`

Reply:
yes